server/middleware/cleanLocks.js

```diff
@@ -0,0 +1,48 @@
+const Cohort = require('../models/cohort.model');
+const List = require('../models/list.model');
+
+const unlockLocks = () => {
+  List.updateMany(
+    { locks: { $exists: true } },
+    {
+      locks: { name: false, description: false }
+    }
+  ).catch(() => {
+    // eslint-disable-next-line no-console
+    console.log('Unlocking locks failed...');
+    process.exitCode(1);
+  });
+
+  List.find({ items: { $exists: true } })
+    .exec()
+    .then(lists => {
+      lists.forEach(list => {
+        const { items } = list;
+
+        items.forEach(item => {
+          const { locks } = item;
+
+          locks.name = false;
+          locks.description = false;
+        });
+
+        list.save();
+      });
+    })
+    .catch(() => {
+      // eslint-disable-next-line no-console
+      console.log('Unlocking locks failed...');
```

Comment:
Please mark specific collection that failed, for example with `console.error('Unlocking list items locks failed')`.